.changeset/plenty-boxes-appear.md

```diff
@@ -0,0 +1,6 @@
+---
+"@directus/storage-driver-cloudinary": patch
+"@directus/storage": patch
+---
+
+Fixed saving edit image when using ROOT with Cloudinary storage
```

Comment:
```suggestion
Fixed an issue where moving files using Cloudinary storage with custom ROOT location would fail. The issue occurred, for example, when trying to save an edited image.
```

Reply:
Should a changeset include "for example"? 🤔
Would not it be very long for the release notes?

Reply:
There's no penalty for having long release notes, so I'm down to make it a little more explicit / explanatory